pkg/clients/wrappers/kyverno/v1alpha2/kyverno_client.go

```diff
@@ -57,6 +57,28 @@ func (c *client) BackgroundScanReports(namespace string) v1alpha2.BackgroundScan
 	}
 }
 
+func (c *client) CleanupPolicies(namespace string) v1alpha2.CleanupPolicyInterface {
+	recorder := metrics.NamespacedClientQueryRecorder(c.metrics, namespace, "CleanupPolicy", metrics.KyvernoClient)
+	return struct {
+		controllerutils.ObjectClient[*kyvernov1alpha2.CleanupPolicy]
+		controllerutils.ListClient[*kyvernov1alpha2.CleanupPolicyList]
+	}{
+		metrics.ObjectClient[*kyvernov1alpha2.CleanupPolicy](recorder, c.inner.CleanupPolicies(namespace)),
+		metrics.ListClient[*kyvernov1alpha2.CleanupPolicyList](recorder, c.inner.CleanupPolicies(namespace)),
+	}
+}
+
+func (c *client) ClusterCleanupPolicies() v1alpha2.ClusterCleanupPolicyInterface {
+	recorder := metrics.NamespacedClientQueryRecorder(c.metrics, "", "CleanupPolicy", metrics.KyvernoClient)
+	return struct {
+		controllerutils.ObjectClient[*kyvernov1alpha2.ClusterCleanupPolicy]
+		controllerutils.ListClient[*kyvernov1alpha2.ClusterCleanupPolicyList]
+	}{
+		metrics.ObjectClient[*kyvernov1alpha2.ClusterCleanupPolicy](recorder, c.inner.ClusterCleanupPolicies()),
+		metrics.ListClient[*kyvernov1alpha2.ClusterCleanupPolicyList](recorder, c.inner.ClusterCleanupPolicies()),
+	}
+}
+
```

Comment:
@eddycharly So, I updated this file but this will give an error because `ObjectClient` does `not implement UpdateStatus` function which is present in the `CleanupPolicy` and `ClusterCleanupPolicy`. So, should we create a new client which will also implement `UpdateStatus` function. Since `ObjectClient` is used for other rest of the CRDs.